src/main/java/org/synyx/urlaubsverwaltung/search/PageStreamSupport.java

```diff
@@ -0,0 +1,23 @@
+package org.synyx.urlaubsverwaltung.search;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Slice;
+
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+public class PageStreamSupport {
```

Comment:
das auch in das extension package?

Reply:
extension-Package der UV oder an sich in die Extension-API? hatte es hier drin, da die Page-Support geschichte in Verbindung mit der Suche hier im Package immer verwendet wurde.

Reply:
ich dachte neben `PersonServiceExtensionImpl` weil es da gebraucht wird.

Reply:
und wenn wir es in einem anderen fachlichen package brauchen, dann hängt das auch von person ab?